modules/virt-configuring-obsolete-cpu-models.adoc

```diff
@@ -21,10 +21,11 @@ metadata:
   namespace: {CNVNamespace}
 spec:
   obsoleteCPUs:
-    cpuModels: <1>
+    cpuModels: <cpuModels>
       - "<obsolete_cpu_1>"
       - "<obsolete_cpu_2>"
-    minCPUModel: "<minimum_cpu_model>" <2>
 ----
-<1> Replace the example values in the `cpuModels` array with obsolete CPU models. Any value that you specify is added to a predefined list of obsolete CPU models. The predefined list is not visible in the CR.
-<2> Replace this value with the minimum CPU model that you want to use for basic CPU features. If you do not specify a value, `Penryn` is used by default.
++
+where:
```

Comment:
If you use the AsciiDoc callout in my suggestion in my other comment above, then these two lines need to be deleted:
```suggestion
```